### ISSUE TITLE
exim4 blacklist by hostname

### ISSUE THREAD

**Issue by alphacabbage1** [BytemarkHosting/symbiosis#8]
The second deny in /etc/exim4/symbiosis.d/10-acl/50-acl-check-rcpt/65-deny-blacklisted-hosts looks like it should be using **+blacklisted_hosts_by_hostname** (and the first might benefit from **!+private_addresses**).

```
  # Similarly, we will deny the message if the host IP is on a local blacklist
  deny    hosts         = +blacklisted_hosts_by_ip
          message       = The IP or hostname used when connecting is locally blacklisted.
          log_message   = IP locally blacklisted
  
  # Similarly, we will deny the message if the hostname is on a local blacklist (and not in an private range)
  deny    hosts         = !+private_addresses : +blacklisted_hosts_by_ip
          message       = The IP or hostname used when connecting is locally blacklisted.
          log_message   = Hostname locally blacklisted

  # OK, now we check whether the sender address is on a local blacklist
  deny    senders       = +blacklisted_senders
          message       = Your email address is locally blacklisted.
          log_message   = Sender locally blacklisted
```
Disclaimer: I'm in the early stages of finding my way around exim. ;)

**Comment by patch0** [comment]
I think this was fixed in 15117a1db910964c0f9f63de8f44e35a0e500a93

Does that look about right?

**Comment by skx** [comment]
Yup
